nle/scripts/plot.py

```diff
@@ -32,6 +32,12 @@
 python -m nle.scripts.plot path/to/run -x 100 -y 50
 ```
 
+Plot a specific run to a specific target column.
+```
+python -m nle.scripts.plot path/to/multiple_runs/2020 -ykey "total_loss"
```

Comment:
This has to be `--ykey` though (two hyphens).

Reply:
you beat me by one second! hit backspace one extra time - squashed the fix commits and pushed

Reply:
No need to worry about the commit, we'll squash this when merging.